lib/src/rules/avoid_positional_boolean_parameters.dart

```diff
@@ -84,6 +88,7 @@ class _Visitor extends SimpleAstVisitor {
   visitMethodDeclaration(MethodDeclaration node) {
     if (!node.isSetter &&
         !node.element.isPrivate &&
+        !_indexEqOp(node) &&
```

Comment:
Seems like we should we just disable this lint for all operators. Consider:
```
bool operator +(bool value) {}
```

Reply:
Great idea!

Reply:
Done. PTAL.